master/internal/webhooks/webhook.go

```diff
@@ -179,3 +183,96 @@ func (t TriggerType) Proto() webhookv1.TriggerType {
 		return webhookv1.TriggerType_TRIGGER_TYPE_UNSPECIFIED
 	}
 }
+
+// Proto returns a proto from a TriggerType.
+func experimentToWebhookPayload(e model.Experiment) *ExperimentPayload {
+	var duration int
+	if e.EndTime != nil {
+		duration = int(e.EndTime.Sub(e.StartTime).Seconds())
+	}
+
+	var slots int
+	if s := e.Config.Resources().Slots(); s != nil {
```

Comment:
1. I thought for an experiment `Slots` would always be empty, and the correct field is `SlotsPerTrial`? 
2. What should be the meaning of this field in payload for multi-trial experiments? slots per trial? slots per trial multiplied by number of trials?

Reply:
Thanks for pointing this out. This value should represent `slotsPerTrial` I have updated a comment in the payload documentation to show this. I am open to making this less ambiguous if we feel the need to.

Reply:
I'd call this field "slots per trial" in the payload as well to avoid the confusion if we ever want to add total slots.